### PR TITLE
Fixed PEP8 violations in code style

### PR DESCRIPTION
**Pull request by osBins**
Formatted the code :
- Imported functions from the concerned modules manually.
- Fixed the conditional statements.
- Reduced line length to 79 characters per line.
- Fixed variable e assignment and defining.
to conform to PEP8 standards.

### CODE REVIEW

File: MarvellOS.py [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+    RIDGE,
+    Entry,
+    DISABLED,
+    RIGHT,
+    Text,
+    PhotoImage,
+    Canvas,
+    StringVar,
+    OptionMenu,
+    TOP,
```
Comment (akshgpt7) [inline MarvellOS.py]: Try to have multiple in one line so as to not use up so many lines for imports.

File: MarvellOS.py [hunk 2]
```diff
@@ -49,11 +64,11 @@ def tick():
...
         clock.config(text=day + "      " + time2 + "  ")
-    if truth == True:
-        try:
-            AFTER = welcome_screen.after(200, tick)
-        except:
-            welcome_screen.after_cancel(AFTER)
+    if truth:
+        #       try:
+        AFTER = welcome_screen.after(200, tick)
+    #       except:
```
Comment (akshgpt7) [inline MarvellOS.py]: You don't have to comment out the except block entirely. Just find the error that it is trying to except, and except that.
If you cannot find that, last resort can be to use at least `except Exception:`.

File: MarvellOS.py [hunk 3]
```diff
@@ -76,11 +91,11 @@ def welcome(o=""):
 
     def show():
         global sh
-        if sh == False:
+        if ~sh:
```
Comment (akshgpt7) [inline MarvellOS.py]: Use `not sh` instead of `~`.

File: MarvellOS.py [hunk 4]
```diff
@@ -105,7 +120,8 @@ def confirm(event=None):
             truth = False
             welcome_screen.after_cancel(AFTER)
             clock = None
-            """Argument "y" tells that welcome screen exists and needs to be destroyed"""
+            """Argument "y" tells that welcome screen exists and needs to be de
+            stroyed"""
```
Comment (akshgpt7) [inline MarvellOS.py]: Just add the entire word to the new line instead of cutting it in the middle.

File: MarvellOS.py [hunk 5]
```diff
@@ -204,16 +226,17 @@ def confirm(event=None):
     )  # clock label
     clock.place(x=0, y=0)
     if truth:
-        try:
-            tick()
-        except:
-            welcome_screen.after_cancel(tick)
+        # try:
+        tick()
+    # except:
```
Comment (akshgpt7) [inline MarvellOS.py]: Again, don't comment out the except block, find the error to except.

File: MarvellOS.py [hunk 6]
```diff
@@ -204,16 +226,17 @@ def confirm(event=None):
...
+    #    welcome_screen.after_cancel(tick)
 
     welcome_screen.mainloop()
 
 
 class Home:
-    """This is the class which makes the app grid screen - If you log in with correct password"""
+    """This is the class which makes the app grid screen - If you log in with c
+    orrect password"""
```
Comment (akshgpt7) [inline MarvellOS.py]: Don't cut the word in the middle

File: MarvellOS.py [hunk 7]
```diff
@@ -341,14 +364,16 @@ def tick1(self):
...
+            )
         if time2 != self.time1:
             self.time1 = time2
             self.clock.config(text=day + "      " + time2 + "  ")
-        try:
+            # try:
             Home.AFTER = home.after(200, self.tick1)
-        except:
-            home.after_cancel(Home.AFTER)
+        # except:
```
Comment (akshgpt7) [inline MarvellOS.py]: Same as above except blocks.

File: MarvellOS.py [hunk 8]
```diff
@@ -496,11 +545,12 @@ def reset(self):
...
-        try:
-            self.t = threading.Thread(target=self.func2)
-            self.t.start()
-        except:
-            pass
+        # try:
+        self.t = threading.Thread(target=self.func2)
+        self.t.start()
+
+    # except:
```
Comment (akshgpt7) [inline MarvellOS.py:541]: Same as above.

File: MarvellOS.py [hunk 9]
```diff
@@ -1176,42 +1312,61 @@ def change_password(self):
...
+            command=self.show,
         )
         self.show.place(x=100, y=150)
         self.show.config(width=15)
         self.co = 1
         self.sh = False
 
     def show(self):
-        if self.sh == False:
+        if ~self.sh:
```
Comment (akshgpt7) [inline MarvellOS.py]: Use `not` instead of `~`.

File: MarvellOS.py [hunk 10]
```diff
@@ -1176,42 +1312,61 @@ def change_password(self):
...
             self.ent["show"] = ""
             self.sh = True
             self.show.config(text="Hide")
-        elif self.sh == True:
+        elif ~self.sh:
```
Comment (akshgpt7) [inline MarvellOS.py]: Again, use `not`.

**Review by akshgpt7 — CHANGES_REQUESTED** [review]
Thanks for your patience on this. Good work.
Just a few changes in the review comments, mostly about `except` blocks.

I'll look into the `self` problem in some time.

**Review by akshgpt7 — CHANGES_REQUESTED** [review]
This is ready to merge, just one small last change:
Make sure all the docstrings are _after_ the function/class definition, and not before them. Eg. line 63. By convention, the docstring is written just after the line containing the `def` keyword. A few functions have it before their definition.

**Comment by osBins** [comment]
I've made the required changes. I request you to let me know if there's something else that needs to be changed/formatted.

**Comment by akshgpt7** [comment]
Thanks for your patience on this. Great work!
This looks ready to merge.

**Pull request merged**